### DISCUSSION TITLE
Enable ui.image to accept Pillow Image object

### DISCUSSION THREAD

**Discussion by nghia-vo** [zauberzeug/nicegui#2067] [Ideas / Feature Requests]
I would like to add more features to a client app ([broh5](https://pypi.org/project/broh5/)) where it needs to display an image (using ui.image) and a plot of histogram or line profile (using ui.pyplot) side by side. Currently ui.image does not accept a numpy array or image object but a string. Could we make ui.image accept Pillow image object too? Some thing like this:

```python
import numpy as np
from PIL import Image
from nicegui import ui

array = np.random.randint(0, 256, (100, 100), dtype=np.uint8)
image = Image.fromarray(array)
ui.image(image)

ui.run()
```
and an option figsize=(4,5) to adjust the figure size.

**Comment by falkoschindler** [comment]
Thanks for the idea, @nghia-vo!
I quickly created PR #2069. What do you think?

**Reply by nghia-vo** [reply]
Looking great! Please merge it. I keen to try :)
